### PR TITLE
Setting up Hound CI

### PR DESCRIPTION
**Pull request by BenieSereine**
**What does this PR do?**

    Setup Hound CI for the project

**Description of Task to be completed?**

    Add .hound.yml file
    Integrate Hound badge into the README file

**How should this be manually tested?**

    N/A

**Any background context you want to provide?**

    N/A

**Screenshots (if appropriate)**
Screenshot from 2020-11-18 10-34-21

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
```
Comment (Niyonsengaeric) [inline .eslintrc.json]: you should well rebase and remove all the conflict!

File: .eslintrc.json [hunk 2]
```diff
@@ -38,4 +39,349 @@
...
+      "args": "after-used"
+    }],
+    "no-use-before-define": 2,
+    "vue/max-len": ["error", {
+      "code": 120,
+      "ignoreHTMLAttributeValues": true
+  }]
+  }
+}
+>>>>>>> be6647a (Visit Africa Project)
```
Comment (tigthor) [inline .eslintrc.json]: Please Rebase

**Review by tigthor — APPROVED** [review]
Approved

File: .eslintrc.json [hunk 3]
```diff
@@ -38,4 +38,4 @@
         }
         ]
     }
-}
```
Comment (Niyonsengaeric) [inline .eslintrc.json:41]: add new line

File: .hound.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+ruby:
+  enabled: true
+  config_file: ".rubocop.yml"
```
Comment (erickyvand) [inline .hound.yml]: You don't need these lines as you will not use Ruby.

File: .hound.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+eslint:
+  enabled: true
+  config_file: ".eslintrc.json"
+tslint:
+  enabled: true
+  config_file: tslint.json
```
Comment (erickyvand) [inline .hound.yml]: You don't need these lines too as you are not going to use TypeScript

File: .hound.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+stylelint:
+  enabled: true
+  config_file: ".stylelintrc.json"
```
Comment (erickyvand) [inline .hound.yml]: Why did you put this?

File: .rubocop.yml [hunk 7]
```diff
@@ -0,0 +1,491 @@
+require: rubocop-rspec
```
Comment (erickyvand) [inline .rubocop.yml]: No need for this file you can delete it

File: .stylelintrc.json [hunk 8]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (erickyvand) [inline .stylelintrc.json]: This file too is not needed

File: tslint.json [hunk 9]
```diff
@@ -0,0 +1,153 @@
+{
```
Comment (erickyvand) [inline tslint.json]: This file too is not needed

**Review by erickyvand — CHANGES_REQUESTED** [review]

**Review by Niyonsengaeric — APPROVED** [review]

**Pull request merged**